cypress/integration/viewer-grid.spec.ts

```diff
@@ -227,7 +234,39 @@ describe("Viewer Grid", () => {
         const calls = stub.getCalls();
 
         expect(calls.map(c => c.args[0]?.detail)).to.deep.equal(expected);
-      })
+      });
+  });
+
+  it('should say the right phrase uppon size change', () => {
+    cy.visit(url);
+
+    prepareComponent();
+
+    cy.get('@root')
+      .find('.dso-map-panel')
+      .invoke('attr', 'aria-live')
+      .should('eq', 'polite')
+      .get('@root')
+      .find('.sizing-buttons span.sr-only')
+      .should('have.text', 'breedte tekstpaneel: breed');
+
+    shrink();
+
+    cy.get('@root')
+      .find('.sizing-buttons span.sr-only')
+      .should('have.text', 'breedte tekstpaneel: middel');
+
+    shrink();
+
+    cy.get('@root')
+      .find('.sizing-buttons span.sr-only')
+      .should('have.text', 'breedte tekstpaneel: smal');
+
+    expand();
+
+    cy.get('@root')
```

Comment:
Je zou een functie `shouldHavePhrase(size: string);` kunnen maken. Als je die dan IN deze `it()` maakt heb je 'n lokale helper functie.